srcpkgs/python-vispy/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'python-vispy'
 pkgname=python-vispy
-version=0.5.3
+version=0.6.1
 revision=1
 archs=noarch
```

Comment:
If the package contains C code, it cannot be noarch any more.

Reply:
Thank you, fixed :) 
Should all python/ruby packages that do not have any C code have archs=noarch ?

Reply:
If they contain pure python or ruby code, then generally yes. I'm not sure if there could be edge cases where there's still architecture dependent stuff in it, though.